src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -88,22 +92,9 @@ export function MessagesList() {
   // We don't want to call `scrollToEnd` again if we are already scolling to the end, because this creates a bit of jank
   // Instead, we use `onMomentumScrollEnd` and this value to determine if we need to start scrolling or not.
   const isMomentumScrolling = useSharedValue(false)
-  const hasInitiallyScrolled = useSharedValue(false)
   const keyboardIsAnimating = useSharedValue(false)
   const layoutHeight = useSharedValue(0)
 
-  // Since we are using the native web methods for scrolling on `List`, we only use the reanimated `scrollTo` on native
-  const scrollToOffset = React.useCallback(
-    (offset: number, animated: boolean) => {
-      if (isWeb) {
-        flatListRef.current?.scrollToOffset({offset, animated})
-      } else {
-        runOnUI(scrollTo)(flatListRef, 0, offset, animated)
-      }
-    },
-    [flatListRef],
-  )
-
```

Comment:
In a galaxy far far away, developers have a Javascript ecosystem that interacts with native code fully synchronously, and they live very happily.

Alas, that is not our galaxy. There's some layouts that are not actually ready yet when we call `scrollTo` right now, particularly around sending a new message and expecting the view to scroll up. We've seen over the past week that the old way of doing things was solid, so let's return to that.

We _will_ still use `scrollTo` for keyboard animation scrolling, because its 🔥🥵🔥 and doesn't cause problems.